application/app.py

```diff
@@ -153,9 +153,19 @@ def game_status():
         else:
             card_ids = [card[0] for card in db.session.query(Round.cardId).filter_by(playerId=player.id, age=game.age, round=game.round).all()]
             cards = Card.query.filter(Card.id.in_(card_ids)).all()
+
+            # the following checks to see if the player has played a card in the next round
+            # which means it is waiting for other players to play this round
+            if Round.query.filter_by(playerId=player.id, round=game.round+1, age=game.age).first():
+                return jsonify(
+                    status="Card Played",
+                    game=print_json(player, players, cards),
+                    players=player_count,
+                    played_card=(Cardhist.query.filter(Cardhist.cardId.in_(card_ids)).first()).serialise()
```

Comment:
Added the played card to the return statement so we can add some popups or animations later.

Reply:
@jacobstringer are you sure first returns the most recent played card or the first?

Reply:
There will only ever be one result as the card_ids will be all the unplayed cards (with only one possibly played since then). Calling first() ensures that the player object is returned, which is needed for the serialise() method.

Although... more than one game could have the same cards, so I better add that check too... didn't think about that.